extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -16,6 +16,9 @@
 
 import {
   CONSENT_ITEM_STATE,
+  CONSENT_STRING_TYPE,
```

Comment:
sanity check please. This won't compile

Reply:
Oops, thanks... was gonna wait for travis to complain :'-)